partner_communication/models/communication_job.py

```diff
@@ -881,14 +875,9 @@ def _print_report(self):
                 try:
                     # Print letter
                     print_name = name[:3] + " " + (job.subject or "")
-                    print_options = job.print_letter(print_name)
-                    output_tray = print_options["output_tray"]
 
-                    # Print attachments in the same output_tray
-                    job.attachment_ids.print_attachments(
-                        output_tray=output_tray,
-                    )
-                    job.write({"state": state, "sent_date": fields.Datetime.now()})
+                    job.with_delay()._print_job_asynchronous(print_name, state)
```

Comment:
```suggestion
                    job.with_delay(channel="root.partner_communication")._print_job_asynchronous(print_name, state)
```

Reply:
Again here I think you can remove the try/except block surrounding this code. The error will be handled by the job itself.